src/NLog/Common/ValueConverter.cs

```diff
@@ -31,20 +31,21 @@
 // THE POSSIBILITY OF SUCH DAMAGE.
 // 
 
-namespace NLog.Config
-{
-    using System;
-    using NLog.Internal;
+using System;
+using NLog.Config;
+using NLog.Internal;
 
+namespace NLog.Common
+{
     /// <summary>
-    /// Default implementation of <see cref="IPropertyTypeConverter"/>
+    /// Convert between types
     /// </summary>
-    internal class PropertyTypeConverter : IPropertyTypeConverter
+    internal class ValueConverter : IPropertyTypeConverter
```

Comment:
Think it should keep its name `PropertyTypeConverter `

Reply:
Well it's used for other Layouts and those aren't really properties. If I also create a IValueConverter, it makes maybe more sense. Anyway, it's internal so PropertyTypeConverter is also fine to me

Reply:
reverted